staging/src/k8s.io/kubectl/pkg/cmd/top/top_pod.go

```diff
@@ -156,17 +159,24 @@ func (o *TopPodOptions) Validate() error {
 			return errors.New("--sort-by accepts only cpu or memory")
 		}
 	}
-	if len(o.ResourceName) > 0 && len(o.Selector) > 0 {
+	if len(o.ResourceName) > 0 && (len(o.LabelSelector) > 0 || len(o.FieldSelector) > 0) {
 		return errors.New("only one of NAME or --selector can be provided")
```

Comment:
Does this message need to be updated to refer to `--field-selector` too?

Reply:
oh yea you are right, I missed it! Thanks

Reply:
@briandorsey , I remember now. I didn't how to rephrase that and I think `field-selector` and `label-selector` are both `selector` so I guess I will change it to `selector`.